xarray/backends/netCDF4_.py

```diff
@@ -251,6 +251,12 @@ def get_dimensions(self):
         return FrozenOrderedDict((k, len(v))
                                  for k, v in iteritems(self.ds.dimensions))
 
+    def get_encoding(self):
```

Comment:
I would lean slightly toward just creating a `get_unlimited_dims` method rather than `get_encoding`, unless we can think of other Dataset wide encodings we might possibly add in the future.